portfolio/src/main/webapp/script.js

```diff
@@ -119,7 +119,13 @@ document.addEventListener('keydown', function(event) {
 
 /**  */
 function sayHello() {
-  fetch('/data').then(response => response.text()).then((helloMessage) => {
-    document.getElementById('hello-container').innerText = helloMessage;
+  fetch('/data').then(response => response.json()).then((helloMessage) => {
```

Comment:
I found a bug :) 

![image](https://user-images.githubusercontent.com/2783959/83467064-bc6d8880-a42d-11ea-86a4-81e9cb412798.png)

Reply:
Thanks for catching that! Definitely not what I intended. Will fix this

Reply:
Done! :)

Reply:
The `Hello!` button is gone from the Mafe_jsonWalkthrough branch, and I don't see it in master, either. It's still hanging around in Mafe_FetchWalkthrough, though. Maybe it got lost in some merge commit?

Reply:
I tried merging one of the previous branches and it did some weird things... I just reverted some of it and will be making a new pull request for the Mafe_jsonWalkthrough branch (the button should be there now) along with a pull request for a branch with the POST form. I’m hoping this brings back to where they should be